### PR TITLE
Move rating stars to the top in the add-on review listing

### LINKED ISSUE #5924: Move rating stars to the top in the add-on review listing
### Describe the problem and steps to reproduce it:

Go to any add-on review listing page, such as https://addons-dev.allizom.org/en-US/firefox/addon/feed-sidebar/reviews/

### What happened?

The star ratings are at the bottom.

### What did you expect to happen?

The UX mock has moved them to the top. Example:

screenshot 2018-08-13 15 18 05

### Anything else we should know?

We are not yet ready to show the helpful / not helpful buttons. See https://github.com/mozilla/addons-frontend/issues/5360

### PR DESCRIPTION
**Pull request by bobsilverberg**
Fixes #5924 

Add-on reviews listing before:

screenshot 2018-08-23 16 33 27

Add-on reviews listing after:

screenshot 2018-08-23 16 32 38

User profile before:

screenshot 2018-08-23 16 35 39

User profile after:

screenshot 2018-08-23 16 36 24

### CODE REVIEW

File: src/amo/components/AddonReviewListItem/styles.scss [hunk 1]
```diff
@@ -15,22 +15,16 @@
...
-  width: 100%;
-
-  @include respond-to(extraExtraLarge) {
-    justify-content: flex-start;
-    margin-top: 0;
-    width: auto;
-  }
+  justify-content: flex-start;
+  margin-top: 0;
+  width: auto;
```
Comment (kumar303) [inline src/amo/components/AddonReviewListItem/styles.scss]: I don't think you need to declare this.

File: src/amo/components/AddonReviewListItem/index.js [hunk 2]
```diff
@@ -192,88 +192,85 @@ export class AddonReviewListItemBase extends React.Component<InternalProps> {
...
-              onClick={this.onClickToBeginReviewReply}
-              className="AddonReviewListItem-begin-reply AddonReviewListItem-control"
+              href="#edit"
+              onClick={this.onClickToEditReview}
+              className="AddonReviewListItem-edit AddonReviewListItem-control"
             >
-              <Icon name="reply-arrow" />
-              {i18n.gettext('Reply to this review')}
+              {isReply
+                ? i18n.gettext('Edit my reply')
```
[the remark this answers is not in the document]
Comment (bobsilverberg) [reply src/amo/components/AddonReviewListItem/index.js]: Oops, sorry. Fixed now.

File: src/amo/components/AddonReviewListItem/styles.scss [hunk 3]
```diff
@@ -15,22 +15,16 @@
...
-  justify-content: flex-end;
-  margin-top: 6px;
-  width: 100%;
-
-  @include respond-to(extraExtraLarge) {
-    justify-content: flex-start;
-    margin-top: 0;
-    width: auto;
-  }
+  justify-content: flex-start;
```
[the remark this answers is not in the document]
Comment (bobsilverberg) [reply src/amo/components/AddonReviewListItem/styles.scss]: I agree, it does look better on the right. I've changed it and we can always change it again later when we have the buttons.

**Comment by bobsilverberg** [comment]
Thanks for the review, @kumar303. All of the issues have been addressed. Here's a new screen shot:

screenshot 2018-08-24 10 00 16

**Comment by bobsilverberg** [comment]
I changed it, but I'm not sure I'm happy with how far to the right the controls are now. What do you think?

screenshot 2018-08-24 11 45 15

**Pull request merged**